### PR TITLE
EN-2319 PAM process list widget fix

### PR DESCRIPTION
**Pull request by alepintus**
Hi @joewhite101 ,
this is the fix for the process list
Could you please have a look?
Thanks!

### CODE REVIEW

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/aps/system/services/kie/api/KieApiManager.java [hunk 1]
```diff
@@ -191,31 +192,38 @@ public void postBpmForm(KieBpmConfig config , final KieApiInputForm form) throws
     }
 
     @Override
-    public JAXBProcessInstanceList processInstancesDataTable(KieBpmConfig bpmConfig, Properties properties) throws Throwable {
+    public JAXBProcessInstanceList processInstancesDataTable(Properties properties) throws Throwable {
+
+        logger.debug("processInstancesDataTable");
```
Comment (joewhite101) [inline plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/aps/system/services/kie/api/KieApiManager.java]: Need to log something meaningful if this is needed

File: plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/apsadmin/portal/specialwidget/BpmProcessDatatableWidgetAction.java [hunk 2]
```diff
@@ -16,10 +39,16 @@
...
         List<KieProcess> processes = getFormManager().getProcessDefinitionsList(config);
+
+
+
         if (!processes.isEmpty()) {
 
             //Horrible hack. To be replaced by an actual call.
             List<String> fields = new ArrayList<>();
+
+            logger.info("***************************** \n"+processes.get(0).toString()+"\n*****************************");
```
Comment (joewhite101) [inline plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/apsadmin/portal/specialwidget/BpmProcessDatatableWidgetAction.java]: Not info.

**Comment by alepintus** [comment]
Thanks for the review.
Logs deleted.

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by alepintus** [comment]
Retest this please

**Comment by entando-service** [comment]
All is Well

**Pull request merged**